File: app/src/main/java/com/xi/android/kantukuang/weibo/AbstractTimelineUrl.java
package com.xi.android.kantukuang.weibo;

import com.google.api.client.http.GenericUrl;
import com.google.api.client.util.Key;


public class AbstractTimelineUrl extends GenericUrl {
    @Key("trim_user")
    public final String trimUser = "1";
    @Key("count")
    public int count = 10;
    @Key("feature")
    public String feature = "2";
    @Key("since_id")
    public String sinceId = null;

    public AbstractTimelineUrl(String encodedUrl) {
        super(encodedUrl);
    }
}

File: app/src/main/scala/com/shen/xi/android/tut/weibo/AbstractTimelineUrl.scala
package com.shen.xi.android.tut.weibo

import com.google.api.client.http.GenericUrl
import com.google.api.client.util.Key


class AbstractTimelineUrl(encodedUrl: String) extends GenericUrl(encodedUrl) {
  @Key("trim_user")
  val trimUser = "1"
  @Key("count")
  var count = 10
  @Key("feature")
  val feature = "2"
  @Key("since_id")
  var sinceId: String = null
}
